argocd-config/overlays/tokyo0/argocd.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: argocd
+  namespace: argocd
+spec:
+  source:
+    targetRevision: HEAD
```

Specify `release` branch. https://github.com/cybozu-go/neco-apps/blob/master/docs/deploy.md#production-cluster

Maybe:
```suggestion
    targetRevision: release
```